tests/components/backup/test_websocket.py

```diff
@@ -380,16 +380,14 @@ async def test_generate_wrong_parameters(
         (
             {
                 "agent_ids": ["backup.local"],
-                "include_addons": ["ssl"],
```

Comment:
Addons and folders are not supported on non Supervisor instance which is tested here.